tools/cli/test/e2e/cli/split_test.go

```diff
@@ -13,35 +13,45 @@ import (
 	"github.com/tufin/oasdiff/diff"
 )
 
-var versions = []string{"2023-01-01", "2023-02-01", "2023-10-01", "2023-11-15", "2024-05-30"}
+var versions = []string{
+	"2023-01-01",
+	"2023-02-01",
+	"2023-10-01",
+	"2023-11-15",
+	"2024-05-30",
+	"2025-01-01",
+}
 
 func TestSplitVersions(t *testing.T) {
 	cliPath := NewBin(t)
-	testCases := map[string]struct {
+	testCases := []struct {
```

Comment:
oasdiff output from the tests - only tags are different because here we are removing unused tags

```shell
=== RUN   TestSplitVersions
=== RUN   TestSplitVersions/Split_filtered_specs_json_dev
tags:
    deleted:
        - Collection Level Metrics
        - Streams


tags:
    deleted:
        - Collection Level Metrics


tags:
    deleted:
        - Collection Level Metrics


=== RUN   TestSplitVersions/Split_filtered_specs_yaml_dev
tags:
    deleted:
        - Collection Level Metrics
        - Streams


tags:
    deleted:
        - Collection Level Metrics


tags:
    deleted:
        - Collection Level Metrics


=== RUN   TestSplitVersions/Split_not-filtered_specs_json_dev
tags:
    deleted:
        - Collection Level Metrics
        - Streams


tags:
    deleted:
        - Collection Level Metrics


tags:
    deleted:
        - Collection Level Metrics


--- PASS: TestSplitVersions (20.86s)
    --- PASS: TestSplitVersions/Split_filtered_specs_json_dev (6.67s)
    --- PASS: TestSplitVersions/Split_filtered_specs_yaml_dev (7.42s)
    --- PASS: TestSplitVersions/Split_not-filtered_specs_json_dev (6.77s)
PASS
ok      github.com/mongodb/openapi/tools/cli/test/e2e/cli       22.822s
```